Sources/OfflineEntitlements/PurchasedProductsFetcher.swift

```diff
@@ -14,9 +14,16 @@
 import Foundation
 import StoreKit
 
+protocol PurchasedProductsFetcherType {
+
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+    func fetchPurchasedProducts() async throws -> [PurchasedSK2Product]
```

Comment:
Why is the void implementation needed? We're already checking API availability before calling `fetchPurchasedProducts`

Reply:
Yeah I'm an idiot, I solved this in a better way so we don't need that void type :)